lambdas/app.py

```diff
@@ -0,0 +1,54 @@
+import json
+import logging
+import os
+
+import sentry_sdk
+from apig_wsgi import make_lambda_handler
+from flask import Flask
+from sentry_sdk.integrations.aws_lambda import AwsLambdaIntegration
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.DEBUG)
+
+env = os.getenv("WORKSPACE")
+if sentry_dsn := os.getenv("SENTRY_DSN"):
+    sentry = sentry_sdk.init(
+        dsn=sentry_dsn,
+        environment=env,
+        integrations=[
+            AwsLambdaIntegration(),
+        ],
+        traces_sample_rate=1.0,
+    )
+    logger.info("Sentry DSN found, exceptions will be sent to Sentry with env=%s", env)
+else:
+    logger.info("No Sentry DSN found, exceptions will not be sent to Sentry")
+
+
+def create_app():
+    app = Flask(__name__)
+
+    @app.route("/")
+    def hello_world():
+        return "Hello, World!"
+
+    @app.route("/about")
+    def about():
+        return "This is the About page."
+
+    return app
+
+
+# this is the function that lambda should use
+apig_wsgi_handler = make_lambda_handler(create_app())
+
+
+def lambda_handler(event: dict, context: dict):
+    # if not os.getenv("WORKSPACE"):
+    #     unset_workspace_error_message = "Required env variable WORKSPACE is not set"
+    #     raise RuntimeError(unset_workspace_error_message)
```

Comment:
Commenting out for now to avoid needing to set environment variable in Docker container (this will be done with Terraform when the time comes). Also commented out the corresponding test in `tests/test_app.py` for the time being.